woocommerce/woocommerce-gutenberg-products-block
# Create withReviews base HOC

This PR creates a `withReviews` HOC so it can be used by `frontend-block.js` and `editor-block.js` as discussed in #841.

#### Accessibility


### How to test the changes in this Pull Request:

1. Create a post with a _Reviews by Product_ block and verify it loads reviews both in the editor and the frontend.
2. In the frontend, interact with the _Order by_ select and the _Load more_ button and verify they still work
